File: Dsa-day119.py
class Solution:
    def maxOperations(self, s: str) -> int:
        count = 0
        oneC = 0
        for i in range(len(s) - 1):
            if s[i] == '1':
                oneC += 1
            if s[i] == '1' and s[i + 1] == '0':
                count += oneC
        return count

File: Dsa-day119.cpp
class Solution {
public:
    int maxOperations(string s) {
        int count = 0, oneC = 0;
        for (int i = 0; i < s.size() - 1; ++i) {
            if (s[i] == '1') oneC++;
            if (s[i] == '1' && s[i + 1] == '0') count += oneC;
        }
        return count;
    }
};
